src/pages/designing/design-resources/index.mdx

```diff
@@ -241,63 +171,13 @@ to help designers with every aspect of image creation and component specs.
 </Column>
 </Row>
 
-## Native mobile
```

Comment:
Another section where these are in-progress. I think we should keep the section an note they are coming soon for Figma.